assets/js/components/notifications/UserInputSuccessNotification.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * UserInputSuccessNotification component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Notification from '../notifications/notification';
+import UserInputSuccessImage from '../../../svg/congrats.svg';
+
+export default function UserInputSuccessNotification() {
+	return (
+		<Notification
+			id="user-input-success"
+			title={ __( 'Congrats! You set your site goals.', 'google-site-kit' ) }
+			description={ __( 'Based on your goals, now Site Kit will begin showing you suggestions how to add more metrics to your dashboard that are relevant specifically to you', 'google-site-kit' ) }
```

Comment:
I think this should have a full-stop at the end.
```suggestion
			description={ __( 'Based on your goals, now Site Kit will begin showing you suggestions how to add more metrics to your dashboard that are relevant specifically to you.', 'google-site-kit' ) }
```

Looking through the code, usage of punctuation isn't 100% consistent, but most instances of `<Notification />` have a `description` prop that is a full sentence. Especially when the title is a full sentence, it'd make sense that this one is too.

I know it wasn't in the IB, but I think that was a small oversight.